gwdetchar/const.py

```diff
@@ -46,7 +46,9 @@
     ('ER9', Segment(1152136817, 1152255617)),
     ('ER10', Segment(1161907217, 1164499217)),
     ('O2', Segment(1164499217, 1187733618)),
-    ('O3', Segment(1187733618, 2000000000)),
+    ('O3', Segment(1187733618, 1269363618)),
+    ('ER15', Segment(1269356418, 1368975618)),
+    ('O4', Segment(1368975618, 2000000000)),
```

Comment:
So the standard we're going with is 'actual end of one run = figurative start of the next run'?  Thus, O3 starts at GPS time 1187733618 = Fri Aug 25 22:00:00 UTC 2017 (actual end of O2 [verified]) and ends at GPS time 1269363618 = Fri Mar 27 17:00:00 UTC 2020 (actual end of O3 [verified]).  I think ER15 should start at GPS time 1269363618 = Fri Mar 27 17:00:00 UTC 2020 then, not GPS time 1269356418 = Fri Mar 27 15:00:00 UTC 2020, which assigns 15:00-17:00 to both O3 and ER15.

Reply:
I see, a bit strange but ok

Reply:
How about O3 runs until ER15 starts. Meaning, 
```
    ('O2', Segment(1164499217, 1187733618)),
    ('O3', Segment(1187733618, 1269363618)),
    ('ER15', Segment(1269363618, 1368975618)),
    ('O4', Segment(1368975618, 2000000000))
```

Reply:
The conventions for epochs in this list are actually quite mixed up.

Reply:
They are a bit mixed up, and it seems odd to me to have to extend O3 forward or ER15 backward to cover time they didn't actually include.  Do we know what actually cares about those epoch time spans?  If this is an urgent PR, I think your current setting, with ER15 starting at the *actual* end of O3 (similar to how the O3 epoch starts at the *actual* end of O2), is fine, and we can ask Duncan and/or open a ticket to decide what it *should* be and if anything needs to be changed.

Reply:
I agree. Not all GPS times are covered by earlier epochs either. I'm ok with epochs starting when we actually start the epoch and allowing the end to run up until the start of the next epoch. We can figure it out later though and make a better choice going forward. Perhaps that was decided before Alex left. I am not sure